augur/measurements/export.py

```diff
@@ -68,8 +68,8 @@ def register_parser(parent_subparsers):
         help="The short label to display for the x-axis that describles the value of the measurements. " +
              "If not provided via config or command line option, the panel's default " +
              f"x-axis label is {DEFAULT_ARGS['x_axis_label']!r}.")
-    config.add_argument("--threshold", type=float,
-        help="A measurements value threshold to be displayed in the measurements panel.")
+    config.add_argument("--threshold", type=float, nargs="+", dest="thresholds",
+        help="Measurements value threshold(s) to be displayed in the measurements panel.")
```

Comment:
It's a little weird to me that the command-line option remains `--threshold` but the JSON field moved to `thresholds`. Regardless of what we decide about the JSON field, can we alias this to `--threshold` and `--thresholds`? We can do so explicitly by specifying both (probably what I'd do), or specify just `--thresholds` and rely on the default prefix-matching of argparse to catch `--threshold` too.

Reply:
Hah, I originally had [both `--threshold` and `--thresholds` options](https://github.com/nextstrain/augur/commit/51c209b3), but then got rid of it because I thought it would be confusing to have both options.

Reply:
Yeah, may be confusing… although I'd think folks would be used to option aliases with the common `-f` / `--foo` pattern of short / long forms.

But maybe it's also clutter and we should specify only `--thresholds`, letting `--threshold` remain by the prefix-matching.

Reply:
Updated in 9eec32679a3353afc7a4f1963dd495ed71c19818